bigbluebutton-html5/private/locales/en.json

```diff
@@ -49,6 +49,7 @@
     "app.presentation.presentationToolbar.fitScreenDesc": "Display the whole slide",
     "app.presentation.presentationToolbar.zoomLabel": "Zoom",
     "app.presentation.presentationToolbar.zoomDesc": "Change the zoom level of the presentation",
+    "app.presentation.presentationToolbar.slide": "Slide {0}",
```

Comment:
and by "slide" you mean?..

how about "skipSlideLabel" ?

Reply:
is not possible change the name for skipSlideLabel because already exist, then i altered for slideLabel.